lib/archethic/contracts/interpreter/scope.ex

```diff
@@ -114,6 +114,32 @@ defmodule Archethic.Contracts.Interpreter.Scope do
     )
   end
 
+  def get_function_ast(function_name, nil) do
```

Comment:
I think args cannot be nil there since the langage force using `function()` with parenthesis that create an ast with en empty list as function arguments